code/scripting/api/objs/particle.cpp

```diff
@@ -196,7 +177,7 @@ ADE_VIRTVAR(AttachedObject, l_Particle, "object", "The object this particle is a
 {
 	particle_h *ph = NULL;
 	object_h *newObj = nullptr;
```

Comment:
Ditto here, with access [here](https://github.com/scp-fs2open/fs2open.github.com/pull/1881/files#diff-90bae49ed40ec7081a0c26a84cfb74b1R191); although this wasn't changed so I might be wrong?

Reply:
Same as above.